### ISSUE TITLE
No swap is not critical, but printed with "critical" style

### ISSUE THREAD

**Issue by lo48576** [greshake/i3status-rust#1040]
# Environment
* Gentoo Linux, amd64
* i3status-rs 0.14.1

# Summary

I have a machine with no swap enabled, and total swap on this machine is (of course) zero bytes.
i3status-rs considers 0MB/0MB swap usage as 100% used, and tells it is "critical".

![screenshot-2021-02-11-193613+0900](https://user-images.githubusercontent.com/1246590/107626029-88c34d80-6ca0-11eb-8c8d-705ac1791949.png)

However, when no swap is enabled, 0MB free swap is no problem, as it is system admin's intention.

# Expected result

At least no swap is not treated as "critical".

I'm not sure what it actually should look like. `0MB/0MB (0%)` would be one option, `0MB/0MB (--%)` also seems good, or simply printing `N/A` or `no swap` would be possible.

# Actual result

Swap usage is treated as 100% used, and it is printed as "critical".

**Comment by MaxVerevkin** [comment]
Your expected result can be achieved with these config entries:
```
format_swap = "N/A"
warning_swap = 101
critical_swap = 101
```

Although there is an option called "clickable" which seems not to work, I'll see what's wrong with it.

**Comment by MaxVerevkin** [comment]
Fixed id #1041 
So if you want to disable swap just add to your config:
```
clickable = false
```
